encoding/mvt/feature.go

```diff
@@ -0,0 +1,606 @@
+package mvt
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/go-spatial/geom"
+	"github.com/go-spatial/geom/encoding/wkt"
+	vectorTile "github.com/go-spatial/geom/encoding/mvt/vector_tile"
+)
+
+var (
+	ErrNilFeature          = fmt.Errorf("feature is nil")
+	ErrUnknownGeometryType = fmt.Errorf("unknown geometry type")
+	ErrNilGeometryType     = fmt.Errorf("geometry is nil")
+)
+
+// TODO: Need to put in validation for the Geometry, as current the system
+// does not check to make sure that the geometry is following the rules as
+// laid out by the spec (i.e. polygons must not have the same start and end
+// point).
+
+// Feature describes a feature of a Layer. A layer will contain multiple features
+// each of which has a geometry describing the interesting thing, and the metadata
+// associated with it.
+type Feature struct {
+	ID       *uint64
+	Tags     map[string]interface{}
+	Geometry geom.Geometry
+}
+
+func wktEncode(g geom.Geometry) string {
+	s, err := wkt.Encode(g)
+	if err != nil {
+		return fmt.Sprintf("encoding error for geom geom, %v", err)
+	}
+
+	return s
+}
+
+func (f Feature) String() string {
+	g := wktEncode(f.Geometry)
```

Comment:
I don't think `wktEncode` needs to be a separate function. I debating over two solutions.
The first one:
use [`wkt.MustEncode()`](https://github.com/go-spatial/geom/blob/master/encoding/wkt/wkt.go#L302) here and delete `wktEncode`. The issue here is that will introduce a panic
if the Geometry is not supported.

Or Two:

To just copy the body of the `wktEncode` into this function. There is no need for wktEncode function.  I think this the only place this is being used. 

Leaning toward the later (solution two).